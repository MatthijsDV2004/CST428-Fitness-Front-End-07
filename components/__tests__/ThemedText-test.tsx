import * as React from 'react';
import renderer, { act } from 'react-test-renderer';

import { ThemedText } from '../ThemedText';

it('renders correctly', () => {
  let tree: renderer.ReactTestRenderer;
  act(() => {
    tree = renderer.create(<ThemedText>Snapshot test!</ThemedText>);
  });

  expect(tree!.toJSON()).toMatchSnapshot();
});
