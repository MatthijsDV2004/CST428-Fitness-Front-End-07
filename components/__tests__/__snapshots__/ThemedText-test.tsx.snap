// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`renders correctly 1`] = `
<Text
  style={
    [
      {
        "color": "#11181C",
      },
      {
        "fontSize": 16,
        "lineHeight": 24,
      },
      undefined,
      undefined,
      undefined,
      undefined,
      undefined,
    ]
  }
>
  Snapshot test!
</Text>
`;
